tracer/test/Datadog.Trace.TestHelpers/EnvironmentHelper.cs

```diff
@@ -477,7 +477,7 @@ public void EnableUnixDomainSockets()
 #endif
         }
 
-        public MockTracerAgent GetMockAgent(bool useStatsD = false)
+        public MockTracerAgent GetMockAgent(bool useStatsD = false, int fixedPort = -1)
```

Comment:
The `-1` works, but a more idiomatic way of doing this is to make `fixedPort` nullable (`int?`), and using a default value `null` to me "there's no fixed port".
```suggestion
        public MockTracerAgent GetMockAgent(bool useStatsD = false, int? fixedPort = null)
```